src/components/BackToTop.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button } from '@blueprintjs/core'
+
+import { useEffect, useState } from 'react'
+
+export const BackToTop = () => {
+  const [showButton, setShowButton] = useState(false)
+  useEffect(() => {
+    const handleScroll = () => {
+      if (window.pageYOffset > 100) {
+        setShowButton(true)
+      } else {
+        setShowButton(false)
+      }
+    }
+    document.addEventListener('scroll', handleScroll)
```

Comment:
```suggestion
    document.addEventListener('scroll', handleScroll, { passive: true })
```

See https://developer.chrome.com/docs/lighthouse/best-practices/uses-passive-event-listeners/

Reply:
Only mouse and touch events need this, the scroll event cannot be default-prevented.

> Lighthouse uses the following process to identify event listeners that may affect scrolling performance:
> 1. Collect all event listeners on the page.
> 2. Filter out non-touch and non-wheel listeners.
> ...